### PR TITLE
Raygun Cross platform error reporting updated 

### PR DESCRIPTION
**Pull request by Apollo9999**
Raygun Cross platform error reporting Detailing Information added

### CODE REVIEW

File: dotnet-free-oss-services.md [hunk 1]
```diff
@@ -44,6 +44,9 @@ Please sort projects alphabetically and provide a one-line description. Where po
 
 * [Bugsnag](https://bugsnag.com/blog/bugsnag-loves-open-source) - Cross platform error reporting.
 * [Sentry](https://sentry.io/for/open-source/) - Cross platform error reporting.
+* [Raygun](https://raygun.com/blog/tags/open-source/) - Cross platform error reporting and Application Performance Monitoring
+
```
Comment (RussKie) [inline dotnet-free-oss-services.md:48]: Please keep the lexicographical order.

**Review by RussKie — APPROVED** [review]

**Comment by Apollo9999** [comment]
Respected Dignatary  @RussKie please approve the Pull Request

File: dotnet-free-oss-services.md [hunk 2]
```diff
@@ -43,6 +44,7 @@ Please sort projects alphabetically and provide a one-line description. Where po
 ## Analytics and Error Reporting
 
 * [Bugsnag](https://bugsnag.com/blog/bugsnag-loves-open-source) - Cross platform error reporting.
+* [Countly](https://count.ly/
```
Comment (RussKie) [inline dotnet-free-oss-services.md:47]: This line is incomplete

File: dotnet-free-oss-services.md [hunk 3]
```diff
@@ -43,8 +44,12 @@ Please sort projects alphabetically and provide a one-line description. Where po
...
+* [Raygun](https://raygun.com/blog/tags/open-source/) - Cross platform error reporting and Application Performance Monitoring.
 * [Sentry](https://sentry.io/for/open-source/) - Cross platform error reporting.
 
+
+
```
Comment (RussKie) [inline dotnet-free-oss-services.md:52]: This can be reverted

File: dotnet-free-oss-services.md [hunk 4]
```diff
@@ -35,6 +35,7 @@ Please sort projects alphabetically and provide a one-line description. Where po
 * [Coverity Scan](https://scan.coverity.com) - Static analysis service to identify defects in C# code.
 * [dotCover](https://www.jetbrains.com/buy/opensource/?product=dotCover) - Code coverage tool integrated with Visual Studio.
 * [.Net Reflector](http://reflectorblog.red-gate.com/2013/07/open-source) - Browse, analyse and decompile IL code.
+* [Sonarqube](https://www.sonarsource.com/open-source-editions/)-static code analysis tools out there. It is an open-source platform for continuous inspection of code quality and performs automatic reviews via static code analysis
```
Comment (RussKie) [inline dotnet-free-oss-services.md:38]:
```suggestion
* [Sonarqube](https://www.sonarsource.com/open-source-editions/) - Static code analysis tools out there. It is an open-source platform for continuous inspection of code quality and performs automatic reviews via static code analysis
```

**Review by RussKie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**